handbook/product.md

```diff
@@ -252,6 +252,7 @@ Premium version supported for bug fixes: **Latest version only**
 Premium support for support/troubleshooting: **All versions**
 
 ### Goals
+At Fleet, our primary quality objectives are *defect reduction* and *customer service*. This entails Key Performance Indicators such as number of bugs resolved per cycle and customer response time, among others. 
```

Comment:
```suggestion
At Fleet, our primary quality objectives are *customer service* and *defect reduction*. This entails Key Performance Indicators such as customer response time and number of bugs resolved per cycle and. 
```

Reply:
As of now "customer service" is the primary objective. Thus, I suggest swapping "customer service" and "defect reduction."